roles/lib_openshift/src/test/unit/conftest.py

```diff
@@ -0,0 +1,55 @@
+import inspect
+
+import pytest
+
+from lib_openshift.library import *  # noqa: F401,F403
+
+
+def pytest_generate_tests(metafunc):
+    if 'binary_lookup_module' in metafunc.fixturenames:
+        lookup_modules = []
+        ids = []
+        for key in list(globals().keys()):
+            value = globals()[key]
```

Comment:
I like that approach, we'd still need a hack like this to show coverage of the "Binary locator" until we can require Ansible 2.3 and leverage module_utils.